charts/lagoon-core/templates/api.deployment.yaml

```diff
@@ -153,14 +155,16 @@ spec:
             secretKeyRef:
               name: {{ include "lagoon-core.keycloak.fullname" . }}
               key: KEYCLOAK_API_CLIENT_SECRET
-        - name: KEYCLOAK_URL
+        - name: KEYCLOAK_FRONTEND_URL
           {{- if .Values.keycloakFrontEndURL }}
```

Comment:
ahh crap, these will need the `/auth` added to them like above

Reply:
https://github.com/uselagoon/lagoon-charts/commit/c84b5b3ae84911e48231aa121aac7cc7fdb0e3e5

i forgot that i was only adding to the init container, not the api :facepalm: bloody templates